openedx/core/djangoapps/schedules/README.rst

```diff
@@ -0,0 +1,366 @@
+Dynamic Pacing Schedules
+========================
+
+The Schedules app allows course teams to automatically email learners in
+self-paced courses. The emails are designed to keep learners engaged
+with a course's content. Learners receive these messages at important
+milestones throughout a course.
+
+With Schedules, the author of a self-paced course opts learners into one
+of two "Schedule Experiences". Learners either receive Weekly Course
+Highlight Messages, or a combination of Recurring Nudges and Upgrade
+Reminders.
+
+The app can send all three message types "out of the box":
+
+-  Recurring Nudges
+-  Upgrade Reminders
+-  Weekly Course Highlight Messages
+
+Recurring Nudges encourage learners to engage with self-paced courses at
+regular intervals. The app sends nudges three days and ten days after a
+learner enrolls in a course.
+
+Upgrade Reminders ask learners to purchase their course's "Verified"
+certificate. The reminders are sent two days from their course's upgrade
+deadline, or two days from the course's end date. Whichever date occurs
+sooner.
+
+Weekly Course Highlight Messages tell learners what to look forward to
+in the coming week of a course. Course authors provide "section
+highlights" when authoring a course in Studio. The app generates emails
+with these section highlights listed in the body of the message.
+
+The app introduces the Schedule object to the edX codebase. Learners
+receive Schedules when they enroll in self-paced courses. With
+Schedules, the app determines when to send particular messages. It
+assumes a steady rate of progress through course materials. It
+determines the cadence of its messaging through a course's "Schedule
+Experience".
+
+Glossary
+--------
+
+-  **Schedule**: Stores the day a learner enrolls in a course and the
+   learner's "upgrade deadline".
```

Comment:
Represents a personalized experience in a self paced course where events happen relative to *your* schedule, not the course's schedule.